pkg/sanity/sanity.go

```diff
@@ -156,6 +156,14 @@ type TestConfig struct {
 	// generator for valid Volume and Node IDs. Defaults to
 	// DefaultIDGenerator.
 	IDGen IDGenerator
+
+	// Repeat count for Volume operations to test idempotency requirements.
+	// some tests can optionally run repeated variants for those Volume operations
+	// that are required to be idempotent, based on this count value.
+	// -1 : Skip idempotency test
+	//  n : repeat n times
+	// Note that missing explicit value leads to default repetition count.
```

Comment:
There isn't really a "missing explicit value" anymore. You can replace that last sentence with "NewTestConfig() by default enables idempotency testing."